src/bp/ui-shared/src/Interface/typings.d.ts

```diff
@@ -17,4 +18,5 @@ export interface ConfirmDialogProps extends ConfirmDialogOptions {
 
 declare module 'botpress/shared' {
   export function confirmDialog(message: string, options: ConfirmDialogOptions): Promise<boolean>
+  export const TooltipStyle: TooltipStyle
```

Comment:
I would prefer if the shared module only exports components or methods. Can you make the tooltip as a shareable components instead ? 

If you really need to have it here, then I suggest creating a namespace "style" which would include "tooltip", and others.

Reply:
I liked the idea of being able to have shared styles that aren't all in the same file just like components to be more modular and lightweight. I will create an actual component for the tooltip as having a css only one isn't giving the result I wanted (will do it in another PR so this one can be merged in case it takes me too long), but I believe in some cases it might be easier to just shared styling to different sections of the app and this might be a good way to do it